package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@api3/chains",
   "license": "MIT",
-  "version": "3.4.1",
+  "version": "3.4.2",
```

Comment:
I'm fine with this change for now, but for the future, can we please keep version bumps outside of PRs? It makes things trickier if there are multiple PRs/branches that we want to include in a given version

Reply:
got it